server/prometheus.js

```diff
@@ -54,8 +127,7 @@ class Prometheus {
      * @param {object} tlsInfo TLS details
      * @returns {void}
      */
-    update(heartbeat, tlsInfo) {
-
+    async update(heartbeat, tlsInfo) {
```

Comment:
Function does not use async.
Also the type-signature would be wrong ^^

```suggestion
    update(heartbeat, tlsInfo) {
```